CONTRIBUTING.md

```diff
@@ -21,21 +21,24 @@ This section describes governance processes we follow in developing Aim.
 
 #### Authors
 
-- Gevorg Soghomonyan ([SGevorg](https://github.com/SGevorg))
+- Gev Soghomonyan ([SGevorg](https://github.com/SGevorg))
 - Gor Arakelyan ([gorarakelyan](https://github.com/gorarakelyan))
 
-#### Core Maintainers
+#### Primary Maintainers
 
 - [alberttorosyan](https://github.com/alberttorosyan)
+- Karen Hambardzumyan ([mahnerak](https://github.com/mahnerak))
+- Ruben Karapetyan ([roubkar](https://github.com/roubkar))
+
+#### Core Maintainers
+
 - [aramaim](https://github.com/aramaim)
 - Arsen Alkamyan ([arsengit](https://github.com/arsengit))
```

Comment:
would be nice to keep the same style here:
FirsName LastName (GH Account link)

Reply:
@alberttorosyan latest changes include that fix